explorer/src/pages/Home/data.tsx

```diff
@@ -448,4 +449,27 @@ export const issuersData: IIssuer[] = [
       },
     ],
   },
+  {
+    name: "Humanode",
+    logo: Humanode,
+    logoDark: Humanode,
+    keywords: ["Proof of Biometric Uniqueness", "Cryptobiometrics"],
+    description:
+      "Humanode is a Layer 1 secured by private biometric verification (cryptobiometrics), ensuring that each node represents a unique, real human. By eliminating reliance on tokens or computational power for validation, Humanode provides robust Sybil resistance, empowering an egalitarian, decentralized ecosystem where governance and participation are driven by human authenticity.",
+    CTALink: "https://humanode.io/",
+    CTATitle: "Go To Humanode",
+    address: "0x4D9D252C6Ef94184b4a2f15Fb682b01294e876A1",
```

Comment:
What is this?

Reply:
This is issuer address

Reply:
Not any one of the contracts? Issuer? This is odd... But ok :)